job/server/src/main/java/alluxio/worker/job/command/JobWorkerHealthReporter.java

```diff
@@ -34,7 +41,17 @@ public JobWorkerHealthReporter() {
    *
    * @return the system load average of the worker
    */
-  public double[] getCpuLoadAverage() {
-    return mProcessor.getSystemLoadAverage(3);
+  public List<Double> getCpuLoadAverage() {
+    return DoubleStream.of(mHardware.getProcessor().getSystemLoadAverage(3)).boxed()
+        .collect(Collectors.toList());
+  }
```

Comment:
is there a reason you wanted to change this now?

Reply:
Nope. I was realizing I'm using List objects in the other metric as well.